autogpt/llm/chat.py

```diff
@@ -40,10 +37,10 @@ def generate_context(prompt, relevant_memory, full_message_history, model):
         create_chat_message(
             "system", f"The current time and date is {time.strftime('%c')}"
         ),
-        # create_chat_message(
-        #     "system",
-        #     f"This reminds you of these events from your past:\n{relevant_memory}\n\n",
-        # ),
+        create_chat_message(
```

Comment:
We should not add  this prompt if the len() of the memory is currently zero.

Reply:
Should be fixed